### ISSUE TITLE
Hot Reload Not Working

### ISSUE THREAD

**Issue by cgtholdingsllc1** [the49ltd/The49.Maui.BottomSheet#43]
This isn't urgent but I wanted to bring up this issue. Not sure if it is possible to fix, but hot reload doesn't work for bottom sheets. This could be a visual studio/Maui issue not recognizing BottomSheet as something that is hot reload compatible.

**Comment by cgtholdingsllc1** [comment]
I figured it wouldn't be an easy thing. Thank you for getting back to me.

**Comment by vhugogarcia** [comment]
It appears this will be possible to implement in .NET MAUI soon 😎. https://github.com/dotnet/maui/pull/16046 

I am not sure, but I wanted to share it 🙂👍🏻

**Comment by paulvarache** [comment]
That is actually great news. It only made it to main yesterday, so it won't be available immediately, but I will definitely look into integrating Hot Reload and Live Visual Tree I upgrade to .net8

**Comment by vhugogarcia** [comment]
> That is actually great news. It only made it to main yesterday, so it won't be available immediately, but I will definitely look into integrating Hot Reload and Live Visual Tree I upgrade to .net8

right! also this may be automatically fixed https://github.com/the49ltd/The49.Maui.BottomSheet/issues/23 or no. hehe!

**Comment by cgtholdingsllc1** [comment]
This is very good news.

**Comment by hansmbakker** [comment]
I guess this is possible now that # is merged
